ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+| __Unnecessary details:__   |  In this particular case the physical threads are an unnecessary detail as they will cause a significant amount of computational load, but have very little effect on the final results. Which details are important to keep and which to delete is dependent on the nature of the model and the analysis.    | - This step can be completed in any CAD program and is more of a general suggestion to be considered continuously during the design phase. <br/></br> - In SpaceClaim one can use a circular pull to flatten the threads off in a similar way to fixing an edge contact. | ![ScrewedUpDetails](https://github.com/NybergWISC/model_development_workflow/blob/master/PicturesSnapshots/ErrorBlocks/DetailCloseUp.JPG "ScrewedUpDetails") |
+
+<details><p><summary>Notes</summary>
+- Most often in real models unnecessary details must be determined intuitively and visually while overlaps, edge contacts, and small gaps cannot be found visually.
+<br/></br>
+- For all of these edits one must work closely with the creator of the model/experiment to determine which complications are important to the end result and which are able to be changed.
+</p></details>
+
+### In Cubit/Trelis
+_Cubit and Trelis are basically the same CAD software with Trelis being the commercial version. They are most useful for the meshing of geometries and assigning materials to volumes._
+
+| Steps  | Graphic User Interface  | In Command Line  |
+|---|---|---|
+| __Import the .sat file into Trelis/Cubit__  | - Can be imported using the GUI with __File -> Import__. <br/></br> - Select the file through file explorer and open. <br/></br> - One important qualifier to include is `[attributes_on]`. | - In the command line the simple form of the command is `import acis [filepath]` <br/></br> - This command can be appended with further commands that can be found by typing `help import acis`. <br/></br> - These qualifiers correspond with the options given when importing files through the GUI. <br/></br> - One important qualifier to include is `[attributes_on]`.  |

```

Comment:
Is the `[attributes on]` qualifier something that gets checked before saying "open"? where/how does one select this qualifier?